lib/Conversion/TritonGPUToLLVM/DotOpToLLVM/MFMA.cpp

```diff
@@ -220,8 +231,16 @@ struct DotOpMFMAConversionHelper {
         return MatrixCoreType::FP32_BF16_BF16_FP32;
       }
     }
-    if (aElemTy.isInteger(8))
-      return MatrixCoreType::INT32_INT8_INT8_INT32;
+    if (aElemTy.isInteger(8)) {
+      auto nonKDim = mfmaEncoding.getNonKDim();
+      auto kWidth = dotOpEncoding.getKWidth();
+      if ((nonKDim == 32 && kWidth == 8) || (nonKDim == 16 && kWidth == 8)) {
+        return MatrixCoreType::INT32_INT8_INT8_INT32_3K;
```

Comment:
Why is `_3K` appended? Is this some kind of convention from the ISA?

Reply:
I saw there is a `FP32_BF16_BF16_FP32_1K` before, and think that is for MI100, so I used a _3K here. Maybe change to `300`?  I am fine with either name.

Reply:
I think `_1K` refers to the faster version of bf16 instruction on MI200. I do not know how it is named. 
@binarman Any thoughts on this?

Reply:
How about use the name `INT32_INT8_INT8_INT32_C3` which indicate it is start from `CDNA 3`. Then we can use `3/4R` for instructions on `RDNA` hardware.

Reply:
`1k` cames from mfma instruction assembly name.
There are two kinds of instructions that works with bf16:
- `V_MFMA_F32_{*}BF16_1K`  available on MI200 and MI300
- `V_MFMA_F32_{*}_BF16` available on MI100, MI200 and MI300

Reply:
Looks like there are no such difference between old int8 and new int 8 instructions, thought.

I like this idea with `INT32_INT8_INT8_INT32_C3`
We can be even more explicit and name it `INT32_INT8_INT8_INT32_CDNA3`

Reply:
changed to `INT32_INT8_INT8_INT32_CDNA3`.